packages/web-app-files/src/mixins/spaces/actions/uploadImage.js

```diff
@@ -38,10 +39,21 @@ export default {
       this.$data.$_uploadImage_selectedSpace = resources[0]
       this.$refs.spaceImageInput.click()
     },
-    $_uploadImage_uploadImageSpace(ev) {
+    $_uploadImage_uploadImageSpace: function (ev) {
```

Comment:
Why this change? I think it's a downgrade hehehehe :trollface:

Reply:
that's weird o_O and I agree with @dschmidt 's initial comment, it's a downgrade 😢